### ISSUE TITLE
[BUG] module 'pinecone' not found with docker install..

### ISSUE THREAD

**Issue by MX10-AC2N** [Kav-K/GPT3Discord#76]
I try the latest docker install but on start there is an error

```Traceback (most recent call last):
File "/bin/gpt3discord", line 7, in <module>
import pinecone
ModuleNotFoundError: No module named 'pinecone'
```

**Comment by MX10-AC2N** [comment]
I use PINECONE_TOKEN="" in .env file

**Comment by Kav-K** [comment]
remove the entire line, instead of using “”, also, I’ve released a fix to the requirements file v4.0.1
